internal/pkg/nodestatus/nodestatus.go

```diff
@@ -33,6 +33,11 @@ import (
 	"sigs.k8s.io/security-profiles-operator/internal/pkg/util"
 )
 
+const (
+	// node name length limit
+	nodeNameLenLimit = 55
```

Comment:
This is a complete nit, but can you change a comment to include that the finalizer can be up to 63 chars in lenght and 55 + len("-deleted") = 63? We might also want to use a constant instead of `-deleted` so that both are next to each other.